src/Swift.Bindings/src/Model/TypeNames/SwiftTypeNameConverter.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace BindingsGeneration;
+
+/// <summary>
+/// Utility class for converting things to Swift type names.
+/// </summary>
+public static class SwiftTypeNameConverter
```

Comment:
Is this layer of abstraction useful?

Reply:
I'm fine with merging this with `SwiftTypeName` if you feel that is too much. The idea was to extract code which is subject to change into separate class.

Reply:
No need, let's review it at the design session.